zivid_camera/src/settings_generator.cpp

```diff
@@ -78,6 +85,12 @@ void writeToFile(const std::string& file_name, const std::string& text)
   }
 }
 
+std::string toUpperCaseFirst(std::string value)
+{
+  value[0] = std::toupper(value[0]);
```

Comment:
Should perhaps check the string is not empty here

Reply:
It is always non-empty as of now, but I can add a check for it.

Reply:
fixed